src/app/models/obj.ts

```diff
@@ -0,0 +1,39 @@
+export interface Obj {
+  id?: number;
+  name?: string;
+  surname?: string;
+  age?: number;
+  dateOfBirth?: string;
+  address?: Address;
+  role?: Role;
+  username?: string;
+  profilePhotoUrl?: string;
+  companies?: Companies[];
```

Comment:
stesso problema di prima, fai un merge locale di develop in angular-esercizio-4 e pusha nuovamente il branch, ti conviene farlo per tutti i branch